OrbitGl/SchedulerTrack.cpp

```diff
@@ -25,127 +25,36 @@ float SchedulerTrack::GetHeight() const {
          layout.GetTrackBottomMargin();
 }
 
-inline Color GetTimerColor(const Timer& timer, TimeGraph* time_graph,
-                           bool is_selected, bool same_tid, bool same_pid,
-                           bool inactive) {
+Color SchedulerTrack::GetTimerColor(const Timer& timer, bool is_selected,
+                                    bool inactive) const {
+  bool is_same_tid_as_selected = timer.m_TID == Capture::GSelectedThreadId;
+  bool is_inactive =
```

Comment:
"inactive" is not used here, causing a compilation error on Linux.

Reply:
Good point, @reichlfl. I essentially followed your suggestion and added a method to determine (in)activeness of timers. Because the logic is easier to follow (IMO), I added a virtual method IsTimerActive() that returns true by default but is implemented differently by SchedulerTrack and ThreadTrack.

Reply:
I actually realized that GpuTrack::IsTimerActive can and should actually follow the same logic that is used in SchedulerTrack::IsTimerActive. When a thread is selected, only show GPU jobs that were submitted from that thread.

Reply:
Thanks! Bit of a nitpick to the changes: Is there a reason you made `IsTimerActive` public? Same for `IsTimerActive` and `SetTimesliceText` (from the base class).

Reply:
That was not intentional. Should be fixed as soon as I push the new changes.